docs/SUMMARY.md

```diff
@@ -39,7 +39,16 @@
     * [Excluding patterns](earthfile/earthignore.md)
 * [The `earthly` command](earthly-command/earthly-command.md)
 * [Configuration reference](earthly-config/earthly-config.md)
-* [CI integration](./ci-integration.md)
+* [CI integration](ci-integration/ci-integration.md)
```

Comment:
I've noticed that people sometimes miss an article if it's the cover article of a section. Wondering if we should instead create a sub-article `ci-integration/overview.md` and put it as the first in the list within this section and leave the top-level `CI integration` not link to anything (gitbook will generate a placeholder page for it).

Reply:
Yeah like I did with the Guides/etc? Sure.